evo-delivery-app/evo-delivery-backend/src/services/route.service.ts

```diff
@@ -1,12 +1,79 @@
-import {DistanceMatrix, Driver, EaEvaluateConfig, EaEvaluateResponse, Order} from "../types";
+import {Depot, DistanceMatrix, Driver, EaEvaluateConfig, EaEvaluateResponse, Order} from "../types";
 import {driverService, draftService, orderService, depotService} from '../services';
 import {googleMatrixClient, eaHttpClientAdapter} from "../clients";
+import driverModel from "../database/models/driver.model";
+import orderModel from "../database/models/order.model";
+import { Console } from "console";
 
 
 export const store = async (draftId: string, routes: EaEvaluateResponse) => {
```

Comment:
I know that this was the method signature, but since we already fetched the drivers/orders data in the draft.evaluate method, we can use it without fetching again the data. 
But keep this method, and make sure it will use the above-described method.

Reply:
done